Makefile

```
@@ -1,29 +1,42 @@
 .PHONY: all dev debug release clean check
```

```suggestion
.PHONY: all dev debug release clean check perf
```
if separate `perf` recipe will be approved